pyomo/solvers/tests/checks/test_gurobi_direct.py

```diff
@@ -22,6 +22,7 @@
 except ImportError:
     gurobipy_available = False
 
+gurobi_available = GurobiDirect().available(exception_flag=False)
```

Comment:
Does this one check the license when you ask about availability? Because if not the changes in this file don't make sense to me.

Reply:
Yes. `gurobipy_available` only checks that the gurobipy python module is importable.  The `gurobi_available` test checks the license.